ddp-workspace/package-lock.json

```diff
@@ -5209,6 +5209,14 @@
         "is-arrayish": "^0.2.1"
       }
     },
+    "error-stack-parser": {
+      "version": "2.0.6",
+      "resolved": "https://registry.npmjs.org/error-stack-parser/-/error-stack-parser-2.0.6.tgz",
+      "integrity": "sha512-d51brTeqC+BHlwF0BhPtcYgF5nlzf9ZZ0ZIUQNZpc9ZB9qw5IJ2diTrBY9jlCJkTLITYPjmiX6OWCwH+fuyNgQ==",
+      "requires": {
+        "stackframe": "^1.1.1"
```

Comment:
Can you pin the versions to an explicit version rather than `^` throughout?  We try to fix versions whenever possible so that we have reproducible builds.

Reply:
Seems to me that it is not worth doing because they are peer-dependencies.
At the same time, all dependencies in our `package.json` are explicitly set.